src/Renci.SshNet.Tests/Common/ArgumentExceptionAssert.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Renci.SshNet.Tests.Common
+{
+    public static class ArgumentExceptionAssert
+    {
+        public static void MessageEquals(string paramName, string expected, ArgumentException exception)
```

Comment:
I'd remove the **paramName** argument and just use **ParamName** from **exception**.  This is about checking **Message**, the value of **ParamName** is (should be) checked separately.

Reply:
I've removed **paramName**.